### DISCUSSION TITLE
não consigo usar o po-page-dynamic-search

### DISCUSSION THREAD

**Discussion by jpinto10** [po-ui/po-angular#1886] [Angular]
Não apresenta, o po-page-dynamic-search.
Como debo proceder?

Apenas essas...: ![image](https://github.com/po-ui/po-angular/assets/29482636/80f5677c-7815-4ac1-9611-7dc061aad7bd)

**Comment by anliben** [comment]
Boa tarde @jpinto10, tudo bem?

Verifique se você instalou o pacote `@po-ui/ng-templates`

Attr.

**Comment by jpinto10** [comment]
Boa tarde.
Ta instalado, tanto que consigo usar, por exemplo o <po-menu>

Eu acredito que haja algum PROBLEMA  NA VERSÃO.. 
Estou usando o Template 15

 

 

De: João Victor ***@***.***> 
Enviada em: terça-feira, 21 de novembro de 2023 16:20
Para: po-ui/po-angular ***@***.***>
Cc: Edilson ***@***.***>; Mention ***@***.***>
Assunto: Re: [po-ui/po-angular] não consigo usar o po-page-dynamic-search (Discussion #1886)

 

Boa tarde @jpinto10 <https://github.com/jpinto10> , tudo bem?

Verifique se você instalou o pacote @po-ui/ng-templates

Attr.

**Comment by anliben** [comment]
Boa Tarde @jpinto10, tudo bem?

Para podermos apoiá-lo precisaremos de mais detalhes para entender a situação

- Utilize o Stackblitz para reproduzir o problema: https://stackblitz.com/edit/po-ui

### Detalhe passo a passo a reprodução da ocorrência, inclua evidências utilizando imagens, vídeos ou gifs:

- Passo 1...*
- Passo 2...
- Detalhe o comportamento atual:
- Detalhe o comportamento que você espera ver:

### Informe a versão dos ambientes:
- PO UI: Ex. v15.8.0
- Angular: Ex. 15
- Node: Ex. 15

### Informe o Browser:
- [ ] Chrome
- [ ]  Firefox
- [ ] Edge
- [ ]  Safari
 - [ ] IE 11
- [ ] Outros

### Informe e o Sistema Operacional:
- [ ] Windows
- [ ] macOS
- [ ] Ubuntu
- [ ] Outros

### Aguardamos seu retorno!
